src/NServiceBus.Msmq.AcceptanceTests/When_distributing_a_command.cs

```diff
@@ -11,103 +11,178 @@
 
     public class When_distributing_a_command : NServiceBusAcceptanceTest
     {
-        static string ReceiverEndpoint => Conventions.EndpointNamingConvention(typeof(Receiver));
+        const int numberOfMessagesToSendPerEndpoint = 20;
+        static string ReceiverAEndpoint => Conventions.EndpointNamingConvention(typeof(ReceiverA));
+        static string ReceiverBEndpoint => Conventions.EndpointNamingConvention(typeof(ReceiverB));
 
         [Test]
         public async Task Should_round_robin()
         {
             var context = await Scenario.Define<Context>()
-                .WithEndpoint<Sender>(b => b.When((session, c) => session.Send(new Request())))
-                .WithEndpoint<Receiver>(b => b.CustomConfig(c =>
+                .WithEndpoint<Sender>(b => b.When((session, c) => session.Send(new RequestA())))
+                .WithEndpoint<ReceiverA>(b => b.CustomConfig(c =>
                 {
                     c.ScaleOut().InstanceDiscriminator("1");
-                    c.GetSettings().Set("Name", "Receiver1");
+                    c.GetSettings().Set("Name", "ReceiverA1");
                 }))
-                .WithEndpoint<Receiver>(b => b.CustomConfig(c =>
+                .WithEndpoint<ReceiverA>(b => b.CustomConfig(c =>
                 {
                     c.ScaleOut().InstanceDiscriminator("2");
-                    c.GetSettings().Set("Name", "Receiver2");
+                    c.GetSettings().Set("Name", "ReceiverA2");
                 }))
-                .Done(c => c.Receiver1TimesCalled > 4 && c.Receiver2TimesCalled > 4)
+                .WithEndpoint<ReceiverB>(b => b.CustomConfig(c =>
+                {
+                    c.ScaleOut().InstanceDiscriminator("1");
+                    c.GetSettings().Set("Name", "ReceiverB1");
```

Comment:
this is related to https://github.com/Particular/NServiceBus/issues/3751 since the scaled out instances have the same name by default.